skimage/data/tests/test_data.py

```diff
@@ -190,3 +190,9 @@ def test_lily_multichannel():
     fetch('data/lily.tif')
     lily = data.lily()
     assert lily.shape == (922, 922, 4)
+
+
+def test_vortex():
+    image_pair = data.vortex()
```

Comment:
I think you need to use the fetch function, so that it gets skipped without internet

Reply:
Ah, thanks @hmaarrfk, I was wondering why all the fetches :joy:

Reply:
btw, this is why I prefer we use some kind of flag/skipif. It's very magical that a test would be skipped depending on the function used inside the test, without any other indications. And for some reason I couldn't run the tests in this module locally — pytest reports no tests are found in this file.

What do you think? Should I try to refactor the data tests? (in a different PR)

Reply:
I think it needs a better decorator name.

The reason I didnt skip outright in the original PR, is that I want to allow developers larger fraction of the tests if they already the data cached

Reply:
> The reason I didnt skip outright in the original PR, is that I want to allow developers larger fraction of the tests if they already the data cached

I mean, this could be done with a sophisticated enough check, right? `@pytest.mark.skipif(file_not_in_cache(fn) and NO_INTERNET)`?